sound/soc/sof/topology.c

```diff
@@ -2474,14 +2475,71 @@ static const struct snd_soc_tplg_ops sof_dspless_tplg_ops = {
 	.bytes_ext_ops_count = ARRAY_SIZE(sof_dspless_bytes_ext_ops),
 };
 
+static const struct card_tplg_ops card_ops[] = {
+	{}
+};
+
 int snd_sof_load_topology(struct snd_soc_component *scomp, const char *file)
 {
 	struct snd_sof_dev *sdev = snd_soc_component_get_drvdata(scomp);
 	const struct firmware *fw;
+	char **tplg_files;
+	int tplg_num = 0;
 	int ret;
+	int i;
 
 	dev_dbg(scomp->dev, "loading topology:%s\n", file);
 
+	for (i = 0; card_ops[i].card_name; i++) {
```

Comment:
since the `card_ops` array is `static`, it means, that you're only going to use it in this file. Then you don't need to use an empty element in it to indicate termination, just use `ARRAY_SIZE()`

Reply:
...I cannot dismiss my own "request-for-change" here... I'll approve it then, just to override it, the other two comments aren't critical, although the one for the array size would be good to address

Reply:
Thanks @lyakh I addressed the other comments. :)